$NetBSD: patch-aj,v 1.2 2007/01/17 02:25:45 rillig Exp $

Tags: not-for-upstream

We don't want xview at all. An additional problem is that on Solaris 10,
there is -lxview, but <xview/xview.h> is missing.

--- configure.orig	2001-07-11 22:58:54.000000000 +0200
+++ configure	2007-01-17 03:12:56.435501068 +0100
@@ -1634,7 +1634,7 @@ int main() {
 xv_main_loop()
 ; return 0; }
 EOF
-if { (eval echo configure:1638: \"$ac_link\") 1>&5; (eval $ac_link) 2>&5; } && test -s conftest${ac_exeext}; then
+if false && { (eval echo configure:1638: \"$ac_link\") 1>&5; (eval $ac_link) 2>&5; } && test -s conftest${ac_exeext}; then
   rm -rf conftest*
   eval "ac_cv_lib_$ac_lib_var=yes"
 else
@@ -1951,6 +1951,11 @@ case "$host_os" in
     OS_LDFLAGS='-L/usr/local/lib'
     OS_LIBCQCAM_OBJS=''
   ;;
+  netbsd*)
+    OS_CFLAGS="-DNETBSD"
+    OS_LDFLAGS="-li386"
+    OS_LIBCQCAM_OBJS=""
+  ;;
   openbsd*)
     OS_CFLAGS='-O2 -DOPENBSD'
     OS_LDFLAGS='-L/usr/local/lib'
